lib/afip_bill/generator.rb

```diff
@@ -11,9 +11,10 @@ class Generator
 
     HEADER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_header.html.erb'.freeze
     FOOTER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_footer.html.erb'.freeze
-    BRAVO_CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B", "11" => "Factura C" }.freeze
+    CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B" }.freeze
```

Comment:
@pipimumu The change that was on master has to remain. So this should be:

```
CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B", "11" => "Factura C" }.freeze
```

Reply:
Please do have a look now @lubc Sir!